### PR TITLE
ES6: Convert sport/football/match-info

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Converts `sport/football/match-info.js` to ES6.

## What is the value of this and can you measure success?

ES6.

### CODE REVIEW

**Comment by SiAdcock** [comment]
Should we also update [this reference](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/bootstraps/enhanced/football.js#L40) to account for the named export?

**Comment by gustavpursche** [comment]
@SiAdcock Of course! Good catch.

File: static/src/javascripts/projects/common/modules/sport/football/match-info.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class MatchInfo {
+    constructor(match: Object, whosCalling: string): void {
+        const base = '/football/api/match-nav';
+        const slug = match.id || [match.date].concat(match.teams).join('/');
+        const page = encodeURIComponent(whosCalling);
+        this.endpoint = `${base}/${slug}.json?page=${page}`;
+    }
+
+    endpoint: string;
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/sport/football/match-info.js]: What are we even exposing this for?

**Review by SiAdcock — APPROVED** [review]
Other than a minor class-related grumble you can happily ignore, this looks great 👍

**Pull request merged**